changelogs/unreleased/6528-Lyndon-Li

```diff
@@ -0,0 +1 @@
+Fix issue #6519. Restrict the client manager of node-agent server to include only Velero resources from the server's namespace, otherwise, the controllers will try to reconcile CRs from all the installed Velero namespaces.
```

Comment:
Do we want upstream changelogs?
We won't be doing anything with them?

Reply:
@kaovilai We don't need them, but it's easier to manage this stuff if it's a clean cherry-pick. This  commit will be removed if/when we rebase to 1.11.2 in the future.